src/features/click-to-load.js

```diff
@@ -589,6 +589,7 @@ function replaceYouTubeCTL (trackingElement, widget) {
         resizeElementToMatch(oldPlaceholder || trackingElement, blockingDialog)
         replaceTrackingElement(widget, trackingElement, blockingDialog)
         showExtraUnblockIfShortPlaceholder(shadowRoot, blockingDialog)
+        hideInfoTextIfNarrowPlaceholder(shadowRoot, blockingDialog, 460)
```

Comment:
We now have a situation where we can trigger both the extra unblock because the placeholder is short, and the shorter copy version. Maybe we can make `showExtraUnblockIfShortPlaceholder()` test for an even shorter height, since the button is visible more often with these changes.

![image](https://user-images.githubusercontent.com/14364481/232484076-0defca4b-c22f-4730-bcf2-651b7867cf29.png)

Reply:
Argh, good point!

I'm not sure how to fix that though, both of these functions aren't ideal. For example, `showExtraUnblockIfShortPlaceholder` doesn't take into account the placeholder's position inside the parent element. [Take the short-placeholder test case](https://privacy-test-pages.glitch.me/privacy-protections/youtube-click-to-load/) where the parent element would be big enough for the placeholder's primary unblock button to be visible, except there is some text above the placeholder.

I'm inclined to leave this as is, and hope it will be sorted out by your refactor. What do you think?

Reply:
My suggestion was just to lower the height that `showExtraUnblockIfShortPlaceholder()` is already checking. But I just noticed this is the same for FB CTL, so it is better if we handle this during the refactor.